.travis.yml

```diff
@@ -35,16 +36,16 @@ notifications:
   email:
       - blacklight-commits@googlegroups.com
 
-global_env:
-  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
-  - ENGINE_CART_RAILS_OPTIONS='--skip-git --skip-listen --skip-spring --skip-keeps --skip-action-cable --skip-coffee --skip-test'
-  - CC_TEST_REPORTER_ID=5042c7358c96b0b926088a4cda3e132fffe7a66ce8047cdb1dc6f0b4b6676b79
-
-jdk: openjdk11
```

Comment:
Don't need JDK anymore if spinning up Solr via Travis's docker infrastructure and our docker-compose configuration.